ooni/scripts/ooniprobe_agent.py

```diff
@@ -121,6 +121,16 @@ def get_running_pidfile():
             return pidfile
     raise NotRunning
 
+def is_stale_pidfile(pidfile):
+    if os.path.exists(pidfile):
+        with open(pidfile) as fd:
+            pid = int(fd.read())
+            return not is_process_running(pid)
+    return False
+
+def get_stale_pidfiles():
+    return [_ for _ in [config.system_pid_path, config.user_pid_path] if is_stale_pidfile(_)]
```

Comment:
Minor nitpick: generally the `_` variable is used for things you don't care about and want to just ignore. In this case I would use any single letter variable of your choosing (`f`?).